app/models/manageiq/providers/redhat/infra_manager/provision/configuration/network.rb

```diff
@@ -68,15 +70,14 @@ def configure_vnic(network_hash, name, vnic)
     network   = find_network_in_cluster(network_hash[:network])
     raise MiqException::MiqProvisionError, "Unable to find specified network: <#{network_hash[:network]}>" if network.nil?
 
-    options = {
-      :name        => name,
-      :interface   => network_hash[:interface],
-      :network_id  => network[:id],
-      :mac_address => mac_addr,
-    }.delete_blanks
-
-    _log.info("with options: <#{options.inspect}>")
-
-    vnic.nil? ? get_provider_destination.create_nic(options) : vnic.apply_options!(options)
+    ems.ovirt_services.configure_vnic(
+      :vm        => destination,
+      :mac_addr  => mac_addr,
```

Comment:
Is `mac_address` required now?  IIRC, if we passed a `nil`, it would try to set it to `nil`.  That was the reason for the `delete_blanks`.